__tests__/db.test.js

```diff
@@ -102,6 +102,24 @@ describe('app', () => {
             expect(body).toEqual([])
           }) 
         })
+        test('POST - api/articles/:article_id/comments - responds with 201 and comment object has been posted to comments array under the correct article_id', () => {
+          const commentToAdd = {
+            username: 'butter_bridge',
+            body: "I haven't posted in a while!"
+          };
+          return request(app)
+          .post('/api/articles/3/comments')
+          .send(commentToAdd)
+          .expect(201)
+          .then((res) => {
+           expect(res.body.comment).toMatchObject({
+              comment_id: 19,
+              body: "I haven't posted in a while!",
+              article_id: 3,
+              author: 'butter_bridge'
+           })
+          })
+        })
```

Comment:
You could add another 201 test that ignores other additional properties that are passed for example:

  **const commentToAdd = {
            username: 'butter_bridge',
            body: "I haven't posted in a while!",
            extraProperty: 'extra property'
          };**